store/aciinfo.go

```diff
@@ -21,46 +21,57 @@ import (
 	"time"
 )
 
+// ACIFetchInfo is used to store information about the fetching process of an ACI.
+type ACIFetchInfo struct {
```

Comment:
> Are you saying that it's not good because it accomplishes the two items you listed instead of having a single responsibility? if so those items are very similar.

Yes, but that's just a personal taste.

> I am not happy about passing ACIFetchInfo to WriteACI so that it can pass it to the constructor. But, a better solution would require a bigger refactoring.

What about:
* inside store/aciinfo.go defined the new variables directly inside the ACIInfo struct (without struct composition).
* define ACIFetchInfo inside store/store.go and use it only as an argument to WriteACI. Then inside WriteACI allocate aciinfo like this: 
```
aciinfo := &ACIInfo{
        BlobKey:                key,
        Name:                   im.Name.String(),
        ImportTime:             time.Now(),
        LastUsed:               time.Now(),
        Latest:                 latest,
        Size:                   sz,
        VerificationHash:       fetchInfo.VerificationHash,
        SourceURL:              fetchInfo.SourceURL,
        InsecureOptions:        fetchInfo.InsecureOptions,
        }
```